doc/user/sql/tail.md

```diff
@@ -19,6 +19,7 @@ Tail will continue to run until cancelled, or until all updates the tailed item
 Field | Use
 ------|-----
 _object&lowbar;name_ | The item you want to tail
+_expression_ | The logical time to tail from onwards
```

Comment:
I like our element names to be a little more descriptive, because we aren't beholden to making this a reusable component in a grammar. wdyt of something like `timestamp_expression`, that gives people a sense of what to do without reading the docs?

Reply:
I'd include the valid value types here rather than the Detail section.